php-fpm/Dockerfile

```diff
@@ -28,23 +27,42 @@ RUN apk add --update \
     php7-tokenizer \
     php7-xml \
     php7-zlib \
-    composer \
     libsodium \
     make \
-    curl
+    curl \
+    shadow \
+    tzdata \
+    sudo \
+    zsh \
+	git \
+	bash
```

Comment:
I've chosen alpine to get a minimal image. But install `bash`, `zsh`, `oh-my-zsh` and all that's stuff is not mandatory. If you want to interact with your container, use `sh`. It's not as sexy as zsh with a cool theme, but I don't need to run that on my production server.